mod_ci/controllers.py

```diff
@@ -1546,8 +1550,7 @@ def set_avg_time(platform, process_type: str, time_taken: int) -> None:
     g.db.commit()
 
 
-def get_info_for_pr_comment(test_id: int) -> PrCommentInfo:
-    """Return info about the given test id for use in a PR comment."""
+def get_subquery_regression_testid_passed(test_id: int):
```

Comment:
Can you please add docstrings for the method?

Reply:
Done